applications/app/AppLoader.scala

```diff
@@ -10,14 +10,14 @@ import conf.CachedHealthCheckLifeCycle
 import conf.switches.SwitchboardLifecycle
 import contentapi.{CapiHttpClient, ContentApiClient, HttpClient, SectionsLookUp, SectionsLookUpLifecycle}
 import controllers._
-import dev.{DevAssetsController, DevParametersHttpRequestHandler}
+import dev.DevAssetsController
 import http.{CommonFilters, CorsHttpErrorHandler}
 import jobs.{SiteMapJob, SiteMapLifecycle}
-import model.{ApplicationContext, ApplicationIdentity}
+import model.ApplicationIdentity
 import services.ophan.SurgingContentAgentLifecycle
 import play.api.ApplicationLoader.Context
 import play.api.BuiltInComponentsFromContext
-import play.api.http.{HttpErrorHandler, HttpRequestHandler}
+import play.api.http.HttpErrorHandler
```

Comment:
Slight inconsistency with the other files:

```suggestion
import play.api.http.{HttpErrorHandler}
```